src/Tasks/Microsoft.NET.Build.Tasks/build/platforms/Windows.targets

```diff
@@ -0,0 +1,23 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup>
+    <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(_ShortFrameworkIdentifier)' == 'uap'">
+    <NugetTargetMoniker>UAP,Version=v10.0</NugetTargetMoniker>
+    <CopyLocalLockFileAssemblies Condition="'$(CopyLocalLockFileAssemblies)' == ''">false</CopyLocalLockFileAssemblies>
+    <TargetPlatformIdentifier>UAP</TargetPlatformIdentifier>
+    <DefineConstants Condition="'$(DisableImplicitFrameworkDefines)' != 'true'">$(DefineConstants);NETFX_CORE;WINDOWS_UWP</DefineConstants>
+    <TargetPlatformVersion Condition="'$(TargetPlatformVersion)' == ''">10.0.10240.0</TargetPlatformVersion>
+    <TargetPlatformMinVersion Condition="'$(TargetPlatformMinVersion)' == ''">10.0.10240.0</TargetPlatformMinVersion>
+    <TargetFrameworkVersion>v5.0</TargetFrameworkVersion>
+    <LanguageTargets>$(MSBuildExtensionsPath)\Microsoft\WindowsXaml\v$(VisualStudioVersion)\Microsoft.Windows.UI.Xaml.$(_SdkLanguageName).targets</LanguageTargets>
```

Comment:
I'd base these: https://github.com/onovotny/MSBuildSdkExtras/blob/be5b6c9c6c1bfe39bf505abcf1a83083a4d6be9f/src/build/netstandard1.0/MSBuildSdkExtras.Common.targets#L8 on the built in properties in the core targets/props.